src/main/python/fx_data.py

```diff
@@ -0,0 +1,47 @@
+import os
+from pathlib import Path
+import ipfx
+from PyQt5.QtCore import QObject, pyqtSignal
+
+from ipfx.bin.run_feature_extraction import run_feature_extraction
+from pre_fx_data import PreFxData
+
+
+class FxData(QObject):
+
+    def __init__(self):
+        super().__init__()
+        self._state_out_of_date: bool = False
+
+    def set_fx_parameters(self,
+                          nwb_path,
+                          ontology_file,
+                          sweep_features,
+                          cell_features,
+                          ):
+
+        self._state_out_of_date = True
+
+        self.input_nwb_file = nwb_path
+        self.stimulus_ontology_file = ontology_file
+        self.output_nwb_file = os.path.join(str(Path.cwd()), "output.nwb")
```

Comment:
per in-person, we should avoid writing out data unless we have a reason to do so. If we must, we should use a temporary directory rather than cwd.